src/app/components/AlertBox/index.tsx

```diff
@@ -12,7 +14,8 @@ interface Props {
   center?: boolean
   /** Example `icon={<Info size="20px" />}` */
   icon?: React.ReactNode
-  children: React.ReactNode
+  children?: React.ReactNode
+  contents?: React.ReactNode
```

Comment:
TODO: Refactor AlertBox, so that `title` prop uses existing styling/structure. And `children` should "replace" the whole contents of the AlertBox.,

Reply:
Opened here: https://github.com/oasisprotocol/oasis-wallet-web/pull/1975